build/pandoc-svg.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
+"""
+Pandoc filter to convert svg files to pdf as suggested at:
+https://github.com/jgm/pandoc/issues/265#issuecomment-27317316
+"""
+
+__author__ = "Jerome Robert"
+
+import mimetypes
+import subprocess
+import os
+import sys
+from pandocfilters import toJSONFilter, Para, Image
```

Comment:
How is `pandocfilters` getting installed? Should it be added to `environment.yml`?